devicetypes/smartthings/zwave-multi-metering-switch.src/zwave-multi-metering-switch.groovy

```diff
@@ -252,20 +267,35 @@ def childOnOff(deviceNetworkId, value) {
 	if (switchId != null) sendHubCommand onOffCmd(value, switchId)
 }
 
+def childOn(deviceNetworkId) {
+	childOnOff(deviceNetworkId, 0xFF)
+}
+
+def childOff(deviceNetworkId) {
+	childOnOff(deviceNetworkId, 0x00)
+}
+
 private onOffCmd(value, endpoint = 1) {
-	delayBetween([
-		encap(zwave.basicV1.basicSet(value: value), endpoint),
-		encap(zwave.basicV1.basicGet(), endpoint),
-		"delay 3000",
-		encap(zwave.meterV3.meterGet(scale: 0), endpoint),
-		encap(zwave.meterV3.meterGet(scale: 2), endpoint)
-	])
+	def cmds = []
+
+	cmds +=	encap(zwave.basicV1.basicSet(value: value), endpoint)
+	cmds += encap(zwave.basicV1.basicGet(), endpoint)
+
+	if (deviceIncludesMeter()) {
+		cmds += "delay 3000"
+		cmds += encap(zwave.meterV3.meterGet(scale: 0), endpoint)
+		cmds += encap(zwave.meterV3.meterGet(scale: 2), endpoint)
+	}
+
+	delayBetween(cmds)
 }
 
-private refreshAll(includeMeterGet = true) {
+private deviceIncludesMeter() {
+	return !isWyfyTouch()
```

Comment:
`!isWYFYTouch()` (you've used wrong letter case)
And please, add this method at the end of the file, not here.

Reply:
Thanks for spotting that error